com.salesforce.ide.ui.editors/src/com/salesforce/ide/ui/editors/apex/assistance/ApexCompletionProcessor.java

```diff
@@ -77,15 +75,6 @@ public boolean updatePresentation(int documentPosition, TextPresentation present
     //  C O N S T R U C T O R S
     public ApexCompletionProcessor() {}
 
-    //   M E T H O D S
```

Comment:
@fdutton This class should be deprecated soon. This is the old mechanism that doesn't work with the new parser.

Was there a reason that you touched this file? Or was it just general cleanup?